egs/wsj/s5/utils/mkgraph.sh

```diff
@@ -169,5 +169,7 @@ cp $lang/phones/silence.csl $dir/phones/ || exit 1;
 cp $lang/phones.txt $dir/ 2> /dev/null # ignore the error if it's not there.
 
 # to make const fst:
-# fstconvert --fst_type=const $dir/HCLG.fst $dir/HCLG_c.fst
```

Comment:
I wasn't thinking that you would make a separate 'const' version of HCLG.fst, since they take up quite a lot of space.  Instead, insert fstconvert into the pipe after add-self-loops, and just make HCLG.fst const.
Of course this will cause a crash unless *all* the programs that read decoding graphs are converted to be able to read regular or const FST.
Bear in mind that there may be other scripts that write HCLG.fsts.scp for purposes like segmentation of training data; those should be converted too, to write const FSTs.